packages/knex/src/schema/SchemaGenerator.ts

```diff
@@ -185,6 +185,9 @@ export class SchemaGenerator {
         const constraintName = meta.collection.includes('.') ? meta.collection.split('.').pop()! + '_pkey' : undefined;
         table.primary(Utils.flatten(meta.primaryKeys.map(prop => meta.properties[prop].fieldNames)), constraintName);
       }
+      if (meta.comment) {
```

Comment:
lets have one blank line here

```suggestion

      if (meta.comment) {
```